build-system/tasks/presubmit-checks.js

```diff
@@ -164,8 +164,8 @@ var forbiddenTerms = {
   'installStorageService': {
     message: privateServiceFactory,
     whitelist: [
-      'extensions/amp-analytics/0.1/amp-analytics.js',
-      'extensions/amp-analytics/0.1/storage-impl.js',
+      'src/runtime.js',
```

Comment:
Do we need to list services that are always installed by runtime for presubmit checks? (I assume no one would manually try to install a service that's already installed through runtime and even if they do, I assume the second install call would be a no-op.)

Anyway, if that's the case, we certainly have some services like 'history` that are not listed here.